### DISCUSSION TITLE
Nested routes

### DISCUSSION THREAD

**Discussion by heriparid** [refinedev/refine#2304] [General]
Hi Team,
Is it possible to define nested routed as it works in react-router-v6?
For example, the routes would be as `/posts/show/:postId/comments/show/:commendId`

My approach is to define the custom-page with the path-URL above in the routes as snippet code below:
```
routerProvider={{
        ...routerProvider,
        routes: [
          {
            path: 'posts/show/:postId/comments/show/:commentId',
            layout: true,
            element: <CommentDetailPage />,
          },
        ],
      }}
```
But I think it's not a good approach, more over when I follow the page, the item menu of "posts" on side menu is not appear as "visited" 
Kindly please help me to solve this problem.
Thanks.

**Comment by omeraplak** [comment]
Hey @heriparid 👋,
It's actually not such a bad idea. Custom Route is a good solution for this. Can you create a Bug issue about `Sider`(Menu)?
